charts/cmc-claim-store/values.template.yaml

```diff
@@ -63,7 +63,7 @@ java:
     FEATURE_TOGGLES_EMAILTOSTAFF: true
     FEATURE_TOGGLES_CCD_ENABLED: false
     FEATURE_TOGGLES_CCD_ASYNC_ENABLED: false
-    DOCUMENT_MANAGEMENT_CASEWORKERROLE: caseworker-cmc
+    DOCUMENT_MANAGEMENT_CITIZENROLE: citizen
```

Comment:
This may no longer be necessary. Please see the comments below. Same applies where ever this is used.